plugins/editor/index.js

```diff
@@ -94,7 +96,8 @@ function editor(app, opts, done){
         'Ctrl-Down': false,
         Tab: false
       });
-      keyExtension.setup(app, codeEditor);
+      keyExtension.setup(app);
```

Comment:
i need to think about this API some more, i'd prefer not having to call `.setup` for this but not sure how to get around it